plugins/pyembed/pyembed.cpp

```diff
@@ -1178,8 +1189,8 @@ class GILBlock
 class PythonRowStream : public CInterfaceOf<IRowStream>
 {
 public:
-    PythonRowStream(PythonThreadContext *_sharedCtx, PyObject *result, IEngineRowAllocator *_resultAllocator)
-    : sharedCtx(_sharedCtx), resultIterator(NULL)
+    PythonRowStream(PyObject *result, IEngineRowAllocator *_resultAllocator)
+    : resultIterator(NULL)
```

Comment:
Because (as the comment says) 
         // NOTE - the caller should already have the GIL lock before creating me